features/product/viewing_products/access_simple_product_show_page_from_products_index.feature

```diff
@@ -0,0 +1,90 @@
+@viewing_products
+Feature: Accessing the simple product show page from product index
+    In order to view detailed product information
+    As a Administrator
+    I want to be able to view product show page
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has "Over sized" and "Standard" shipping category
+        And the store has a tax category "No tax" with a code "nt"
+        And the store classifies its products as "Shield" and "Equipment"
+        And the store has a product "Iron shield" priced at "$20.00"
+        And the product "Iron shield" has orginal price "$25.00"
+        And there are 4 units of product "Iron shield" available in the inventory
+        And the "Iron shield" product has an image "mugs.jpg" with "main" type
+        And product "Iron shield" have the code equals "123456789"
```

Comment:
`And product "Iron shield" code equals "123456789"` or `And product "Iron shield" has the code that equals "123456789"`

However, I would vote for not using the word `equals`, since it's a bit technical.